cli/tests/testdata/inline_js_source_map_2_with_inline_contents.js.out

```diff
@@ -1,4 +1,2 @@
 error: Uncaught Error: Hello world!
-throw new Error("Hello world!" as unknown as string);
-      ^
```

Comment:
Source lines for inline user source maps are eliminated for now as a side effect of this cleanup, but currently only work in very specific arbitrary cases and aren't important. I think this 'regression' is fine considering.

Reply:
What do you mean by inline source maps?

Reply:
`//# sourceMappingURL=` at the bottom. Currently if that source map additionally contains the inline source in the `"sources"` field, and the top stack frame is the one selected for the source line, you would get a source line. You don't anymore with this change which isn't a big deal.

Reply:
Yeah I guess that's fine.